packages/core/src/utils/index.ts

```diff
@@ -1,10 +1,9 @@
-export * from './escape'
 export * from './object'
-export * from './basic'
 export * from './helpers'
 export * from './map'
 export * from './countable-set'
 export * from './layer'
 export * from './variant-group'
 export * from './warn'
 export * from './handlers'
+export * from '@unocss/utils'
```

Comment:
We don't need to export it, I think. 🤔

Reply:
> It's a huge amount of work. 

I can help you do this.

> I don't think this is the time.

I dont't know what's your worried about? You mean, these changes will break the previous usage, right?
If true, 
Currently, most of the utility functions of `@unocss/core` and `@unocss/mini` are used by `preset` developers or others, but it do not affect the use of simple user. Now that `@unocss/utils` exists, we should require developers to import it correctly in new versions.

Reply:
Done